packages/caip/src/adapters/banxa/index.ts

```diff
@@ -47,3 +50,18 @@ export const getSupportedBanxaAssets = () =>
     CAIP19,
     ticker
   }))
+
+const caip2ToBanxaBlockchainCodeMap: Record<CAIP2, string> = {
+  'eip155:1': 'ETH',
+  'bip122:000000000019d6689c085ae165831e93': 'BTC',
+  'cosmos:cosmoshub-4': 'COSMOS'
+}
+
+export const getBanxaBlockchainFromBanxaAssetTicker = (asset: string): string => {
+  const assetCAIP19 = banxaTickerToCAIP19(asset.toLowerCase())
```

Comment:
```suggestion
```suggestion
/**
 * Convert a banxa asset identifier to a Banxa chain identifier for use in Banxa HTTP URLs
 *
 * @param {string} banxaAssetId - a Banxa asset string referencing a specific asset; e.g., 'atom'
 * @returns {string} - a Banxa chain identifier; e.g., 'cosmos'
 */
export const getBanxaBlockchainFromBanxaAssetTicker = (banxaAssetId: string): string => {
  const assetCAIP19 = banxaTickerToCAIP19(banxaAssetId.toLowerCase())
```